HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -1069,6 +1069,8 @@
       <sch:assert role='ERROR' test='count(h:FractionHeatLoadServed) = 1 or count(../h:HeatPump/h:BackupSystem) &gt;= 1'>Expected 1 element(s) for xpath: FractionHeatLoadServed</sch:assert> <!-- See [HeatingSystem=HeatPumpBackup] -->
       <sch:assert role='ERROR' test='count(h:extension/h:FanPowerWattsPerCFM) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/FanPowerWattsPerCFM</sch:assert>
       <sch:assert role='ERROR' test='number(h:extension/h:FanPowerWattsPerCFM) &gt;= 0 or not(h:extension/h:FanPowerWattsPerCFM)'>Expected extension/FanPowerWattsPerCFM to be greater than or equal to 0</sch:assert>
+      <sch:assert role='ERROR' test='count(h:extension/h:FanModelType) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/FanModelType</sch:assert>
+      <sch:assert role='ERROR' test='h:extension/h:FanModelType[text()="psc" or text()="bpm"] or not(h:extension/h:FanModelType)'>Expected extension/FanModelType to be 'psc' or 'bpm'</sch:assert>
```

Comment:
@yzhou601 This should be FanMotorType, not FanModelType. Please update this as as well as the hpxml.rb variable names and documentation.

Reply:
Oops, good catch!